pots/migrations/0008_migrate_ft_to_token.py

```
@@ -0,0 +1,23 @@
+    Token = apps.get_model("tokens", "Token")
+    for payout in PotPayout.objects.all():
+        token = Token.objects.get(id=payout.ft)
+        payout.token = token
+        payout.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("pots", "0007_potpayout_token"),
+    ]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)